platform/ios/ios.cmake

```diff
@@ -168,27 +165,113 @@ if(MBGL_IOS_RENDER_TEST)
             mbgl-vendor-zip-archive
     )
 
+    set_target_properties(RenderTestApp PROPERTIES XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "")
+    set_target_properties(RenderTestApp PROPERTIES XCODE_ATTRIBUTE_CODE_SIGNING_REQUIRED "NO")
+
     find_package(XCTest REQUIRED)
 
     xctest_add_bundle(RenderTestAppTests RenderTestApp ${PROJECT_SOURCE_DIR}/render-test/ios/tests/Tests.m)
 
-    set_target_properties(
-        RenderTestAppTests
-        PROPERTIES
-            XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET
-            "${IOS_DEPLOYMENT_TARGET}"
-            XCODE_ATTRIBUTE_ONLY_ACTIVE_ARCH
-            $<$<CONFIG:Debug>:YES>
-    )
-
     target_include_directories(
         RenderTestAppTests
         PUBLIC ${PROJECT_SOURCE_DIR}/render-test/ios
     )
 
     xctest_add_test(XCTest.RenderTestApp RenderTestAppTests)
 
+    set_target_properties(RenderTestAppTests PROPERTIES XCODE_ATTRIBUTE_IPHONEOS_DEPLOYMENT_TARGET "${IOS_DEPLOYMENT_TARGET}")
+    set_target_properties(RenderTestAppTests PROPERTIES XCODE_ATTRIBUTE_ONLY_ACTIVE_ARCH $<$<CONFIG:Debug>:YES>)
     set_target_properties(RenderTestAppTests PROPERTIES MACOSX_BUNDLE_INFO_PLIST ${PROJECT_SOURCE_DIR}/render-test/ios/tests/Info.plist)
+    set_target_properties(RenderTestAppTests PROPERTIES XCODE_ATTRIBUTE_CODE_SIGN_IDENTITY "")
+    set_target_properties(RenderTestAppTests PROPERTIES XCODE_ATTRIBUTE_CODE_SIGNING_REQUIRED "NO")
+endif()
+
+if(MBGL_IOS_UNIT_TEST)
+    execute_process(COMMAND ditto ${PROJECT_SOURCE_DIR}/test/fixtures ${CMAKE_CURRENT_BINARY_DIR}/test-data/test/fixtures)
+    execute_process(
+        COMMAND
+            ditto ${PROJECT_SOURCE_DIR}/mapbox-gl-js/src/style-spec/reference
+            ${CMAKE_CURRENT_BINARY_DIR}/test-data/mapbox-gl-js/src/style-spec/reference
+    )
+
+    set(
+        RESOURCES
+        ${PROJECT_SOURCE_DIR}/platform/ios/test/common/Main.storyboard
+        ${PROJECT_SOURCE_DIR}/platform/ios/test/common/LaunchScreen.storyboard
+        ${CMAKE_CURRENT_BINARY_DIR}/test-data/test
+        ${CMAKE_CURRENT_BINARY_DIR}/test-data/mapbox-gl-js
```

Comment:
Can we use `${CMAKE_CURRENT_BINARY_DIR}/test-data` for all subfolders?

Reply:
No, because the test expects that the resources are in <basePath>/test and <basePath>/mapbox-gl-js. If I use `${CMAKE_CURRENT_BINARY_DIR}/test-data` then the files would be in <basePath>/test-data/test and they would not be found.

Reply:
Yes, changed the cmake and code accordingly.